core/config/general_config.go

```diff
@@ -42,9 +42,9 @@ var (
 
 type GeneralOnlyConfig interface {
 	Validate() error
-
-	SetLogLevel(zapcore.Level) error
-	SetLogSQL(bool)
+	SetDialect(dialects.DialectName)
```

Comment:
Is this used?

Reply:
Nope, this was a bad merge conflict resolution. Will remove :+1: